synapsis/src/main/java/com/soen/synapsis/appuser/AppUserController.java

```diff
@@ -1,12 +1,10 @@
 package com.soen.synapsis.appuser;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
```

Comment:
You seem to be working off a very old version of the code, which I think is why all these imports are removed. I'm pretty sure if we merged this, our compilation would break. Could you merge main into your branch and make sure everything still works?

Reply:
It shouldn't cause any issues.